src/Mvc/Mvc.Razor.RuntimeCompilation/src/IAssemblyPartResolver.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using Microsoft.AspNetCore.Mvc.ApplicationParts;
+
+namespace Microsoft.AspNetCore.Mvc.Razor.RuntimeCompilation
+{
+    internal interface IAssemblyPartResolver
```

Comment:
Why is there a public and non-public interface of the same name?

Reply:
Originally i thought this might make it into 3.0 so i figured it'd be best not to introduce new public API's. I suppose if it goes into 3.1 it would be ok if this were public. If this is open for review and not deleted i suppose the commit chain is broken?

Reply:
Ok. The interface is now public again and added to options, so it can be extended.